sculture-rest/src/main/java/sculture/models/response/BaseStoryResponse.java

```diff
@@ -40,110 +33,60 @@ public void setId(long id) {
     private long report_count;
 
 
-    public BaseStoryResponse(Story story, List<String> tags, String owner_username, String editor_username) {
+    public BaseStoryResponse(Story story, TagStoryDao tagStoryDao, UserDao userDao) {
```

Comment:
Setting usernames and tags are moved to the Response class. The dao classes are provided for retrieval. I have tried to access dao classes from response model without giving as a parameter but I failed.